packages/flutter/test/gestures/double_tap_test.dart

```diff
@@ -615,6 +567,7 @@ void main() {
     expect(doubleTapRecognized, isTrue);
     GestureBinding.instance.gestureArena.sweep(5);
     expect(doubleTapRecognized, isTrue);
+    expect(tapUp2.globalPosition, up5.position);
 
     tap.dispose();
   });
```

Comment:
I this is supposed to allow "double tap then drag", then maybe it would be worth it to include a test that covers that gesture.  Assuming it's a new gesture and none of these existing tests cover it.